src/main/java/com/vaadin/framework8/migrate/Migrate.java

```diff
@@ -182,9 +182,9 @@ private static String modifyDeclarative(String htmlFile, String version) {
 
             // Version
             htmlFile = htmlFile.replaceAll(
-                    "<meta name=\"vaadin-version\" content=\"7.*\">",
-                    "<meta name=\"vaadin-version\" content=\"" + version
-                            + "\">");
+                    "<meta(.*)name=\"vaadin-version\"(.*)content=\"7.*\"(.*)>",
+                    "<meta$1name=\"vaadin-version\"$2content=\"" + version
```

Comment:
I think you can ignore non-standard stuff in metas when you write it because in any way we will overwrite the design file meta into
<meta name="vaadin-version" content="8.0.0"/>